package NSkobkaInfo;

message TSkobkaInfo {
    optional float DocFreq = 1;
    optional float DocFreqLnk = 2;
    optional uint64 ApproxHitCount = 3;
    optional bytes Text = 4;
    optional string Lemma = 5;
    optional bool LemmaIsBastard = 6;
    optional uint32 LemmaQuality = 7;
    optional bool IsMultitoken = 8;
    optional bool LemmaIsSubstantive = 9;
    optional bool LemmaIsVerb = 10;
    optional bool LemmaIsAdjective = 11;
    optional bool LemmaIsGeo = 12;
    optional bool IsGeoCity = 13;
    optional bool IsGeoAddr = 14;
    optional uint32 NumLemmas = 15;
    optional uint32 NumForms = 16;
    optional float WaresWeight = 17;
}
